spec/mailers/notification_mailer_spec.rb

```diff
@@ -97,10 +103,153 @@
       let(:type) { 'new_ally_request' }
 
       subject(:email) { NotificationMailer.notification_email(recipient, data) }
+      # subject(:email) { NotificationMailer.notification_email(1, data) }
```

Comment:
Any reason why this is commented out? Just WIP stuff?

Reply:
Yeah, this was WIP stuff for debugging the tests. I've removed it in a new commit.